test/TEST-13-NSPAWN-SMOKE/test.sh

```diff
@@ -15,6 +15,7 @@ check_result_qemu() {
     umount $TESTDIR/root
     [[ -f $TESTDIR/failed ]] && cat $TESTDIR/failed
     ls -l $TESTDIR/journal/*/*.journal
+    journalctl -u testsuite --no-pager --file=$TESTDIR/journal/*/*.journal
```

Comment:
> There is https://anonscm.debian.org/cgit/pkg-systemd/systemd.git/commit/debian/tests/upstream?id=063876a7297ef0ae17ccc697667b0cce0e9e4880

Hm, actually, we should print all journalfiles (not only `system.journal`)

For example, `TEST-10-ISSUE-2467` [failed](https://objectstorage.prodstack4-5.canonical.com/v1/AUTH_77e2ada1e7a84929a74ba3b87153c0ac/autopkgtest-xenial-pitti-systemd-semaphore/xenial/amd64/s/systemd-upstream/20161114_201202_06ec4@/log.gz):
```
-rw-r-----+ 1 root systemd-journal 4845568 Nov 14 20:03 /var/tmp/systemd-test.dvMn7p/journal/a0d16806410a41ce97043a81d131dbda/system.journal
-rw-r-----+ 1 root systemd-journal 4845568 Nov 14 20:03 /var/tmp/systemd-test.dvMn7p/journal/a0d16806410a41ce97043a81d131dbda/system@7ce09fc9ab6d452aab1e77842927a3ce-0000000000000001-0005414852ee24c6.journal
TEST RUN: https://github.com/systemd/systemd/issues/2467 [FAILED]
Makefile:8: recipe for target 'run' failed
make: Leaving directory '/tmp/autopkgtest.Dw5D5f/build.zhW/systemd-debian/test/TEST-10-ISSUE-2467'
---- /var/tmp/systemd-test.dvMn7p/journal/a0d16806410a41ce97043a81d131dbda/system.journal ----
```
But there are not required data (`system@7ce09fc9ab6d452aab1e77842927a3ce-0000000000000001-0005414852ee24c6.journal`)